pkg/composableschemadsl/parser/tests/caveatstype.zed.expected

```diff
@@ -35,7 +35,7 @@ NodeTypeFile
                       caveat-name = somecaveat
                       end-rune = 67
                       input-source = caveats type test
-                      start-rune = 53
+                      start-rune = 58
```

Comment:
There's a number of these changes where `with` is now on the other side of a node boundary. This results in the `start-rune` being incremented by 5, which is consistent with #2141.